client/src/app/app.component.ts

```diff
@@ -54,22 +54,24 @@ export class AppComponent {
      * TODO: Overloading can be extended to more functions.
      */
     private overloadArrayToString(): void {
-        Array.prototype.toString = function(): string {
```

Comment:
I checked and this was not a problem to comment in. Please verify

Reply:
Okay, with your (slightly different) ways of assembling a list we indeed avoid this problem; I am still unhappy with sneakily overwriting natiive functions.